docs/source/docker/getting_started/setup.rst

```diff
@@ -119,20 +154,45 @@ You should see an output similar to this one:
     [2022-05-02 20:37:27] Lightly Docker Solution v2.2.0
     [2022-05-02 20:37:27] Congratulations! It looks like the Lightly container is running!
 
-Head on to :ref:`rst-docker-first-steps`  to see how to sample your dataset!
+
+Start the Lightly Worker
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+**Finally**, start the docker in worker mode. In this mode, the docker will long-poll
+the Lightly API for new jobs to process. To do so, a worker needs to be registered.
 
 
-Update Lightly Docker
-^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+.. note:: You only have to register each worker once. The registry is required because
+    it's possible to have several workers registered at the same time working on different
+    jobs in parallel.
 
-To update the docker we simply need to pull the latest docker image.
+.. code-block:: python
+
+    # execute the following code once to get a worker_id
+    from lightly.api import ApiWorkflowClient
+
+    client = ApiWorkflowClient(token='MY_AWESOME_TOKEN') # replace this by your token
+    worker_id = client.register_compute_worker()
+    print(worker_id)
+
+Store the `worker_id` in a secure location and then start the docker in worker mode with
```

Comment:
Maybe add a link to the web app to see the list of registered workers? (https://app.lightly.ai/compute/workers)